client/state/posts/reducer.js

```diff
@@ -261,7 +263,16 @@ export function edits( state = {}, action ) {
 				}
 			} );
 
-		case POST_EDITS_RESET:
+		case EDITOR_START:
+			return Object.assign( {}, state, {
+				[ action.siteId ]: {
+					...state[ action.siteId ],
+					[ action.postId || '' ]: { type: action.postType }
+				}
+			} );
+
+		case EDITOR_STOP:
+		case POST_SAVE_SUCCESS:
```

Comment:
It is an edit in that it needs to be sent with the initial payload for a new draft of non-post types. Though you're right it's not really editable for saved items and posts of type "post".